ReleaseProcess.md

```diff
@@ -0,0 +1,24 @@
+# Release process for OrderlyWeb
+
+1. Get the version of the OrderlyWeb repo that you want to release locally (typically the latest master branch).  
+1. Tag and build the release log by running 
+   `./scripts/release/make-release.py`
+   This script will warn you if any tickets are merged in that are not "Ready 
+   to Deploy". It will prompt you to push the tag to git and to tag and push Docker images by running the tag-images 
+   script.
+1. Check build has passed in Teamcity
+1. Deploy to the support machine(s), test and deploy to production when ready (see below).
+1. Use `RELEASE_LOG.md` to know which tickets to update to the 'Deployed' status
```

Comment:
In fact the script should update the tickets' "fixed in build" field with the release number (see e.g. https://vimc.myjetbrains.com/youtrack/issue/VIMC-2957) It's possible we haven't added this custom field to the mrc project tickets? If not we should do, it's v useful!